### ISSUE TITLE
vertical_parameter_alignment is triggered for @ViewBuilder

### ISSUE THREAD

**Issue by Igor-Palaguta** [realm/SwiftLint#2792]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

`vertical_parameter_alignment` rule is triggered for this case:

```swift
    init(data: Data,
         numberOfColumns: Int,
         spacing: Length,
         @ViewBuilder content: @escaping (Data.Element.IdentifiedValue) -> Content) {
        self.grid = Grid(items: data, numberOfColumns: numberOfColumns)
        self.spacing = spacing
        self.content = content
    }
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
0.33.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)?
downloaded package file from github

```swift
// This triggers a violation:
    init(data: Data,
         numberOfColumns: Int,
         spacing: Length,
         @ViewBuilder content: @escaping (Data.Element.IdentifiedValue) -> Content) {
        self.grid = Grid(items: data, numberOfColumns: numberOfColumns)
        self.spacing = spacing
        self.content = content
    }
```

Can perform PR if agreed

**Comment by Igor-Palaguta** [comment]
@marcelofabri  Thank you for fixing this, but now warning is not triggered in Xcode 11, but is triggered in Xcode 10,
This code under conditional compilation using `#if canImport(SwiftUI)`

**Comment by marcelofabri** [comment]
@Igor-Palaguta unfortunately, SourceKit from Xcode 10.2 doesn’t understand attributes in function parameters, so the information is not available

**Comment by Igor-Palaguta** [comment]
Is there way to disable swiftlint for specific xcode?

**Comment by jpsim** [comment]
The "Disabling rules in code" section of the readme might be helpful: https://github.com/realm/SwiftLint#disable-rules-in-code
